iotdb-client/client-py/release.sh

```diff
@@ -27,7 +27,7 @@ rm -Rf dist
 rm -Rf iotdb_session.egg_info
 
 # (Re-)build generated code
-(cd ../..; mvn clean package -pl iotdb-client/client-py -am)
```

Comment:
This shell script is only used for releasing python package, not in the CI I believe. The user should be a RM, basically me...